### ISSUE TITLE
Use of colon in METS identifiers makes XML schema validation fail

### ISSUE THREAD

**Issue by hsilva-keep** [DLMArchivalStandardsBoard/CommonSpecification#2]
### Requirement from section 5.2 of the document that makes XML schema validation fail 
The ID data type in XML does by default not allow for identifiers which start with a number. To
overcome this limitation and in order to allow for interoperable package identification all
identifiers within Common Specification metadata MUST start with an identifier prefix, followed by
a colon, and the actual value of the identifier.

Example: OBJID="UUID:5d378f86-28a1-41d8-a2b9-264b10fbd511"

### Evidence
Using java:
```
org.xml.sax.SAXParseException; systemId: file:/tmp/temp3503566096366743320/METS.xml; lineNumber: 11; columnNumber: 60; cvc-datatype-valid.1.2.1: 'UUID:B5B90536-D4C6-4A72-81E2-F4568BBF2370' is not a valid value for 'NCName'.
```

Using bash:
```bash
$> xmllint --noout METS.xml --schema schemas/mets.xsd 
METS.xml:13: element dmdSec: Schemas validity error : Element '{http://www.loc.gov/METS/}dmdSec', attribute 'ID': 'UUID:B41AA4B3-17D1-4D89-83B2-17420F43D85B' is not a valid value of the atomic type 'xs:ID'.
METS.xml:14: element mdRef: Schemas validity error : Element '{http://www.loc.gov/METS/}mdRef', attribute 'ID': 'UUID:CD838CE3-F467-4828-88EE-6A499EE6C160' is not a valid value of the atomic type 'xs:ID'.
METS.xml:16: element dmdSec: Schemas validity error : Element '{http://www.loc.gov/METS/}dmdSec', attribute 'ID': 'UUID:4847F90C-AC60-4D55-A579-2ECF736C33C6' is not a valid value of the atomic type 'xs:ID'.
...
```

### Reference documentation
- http://www.datypic.com/sc/xsd/t-xsd_ID.html
- http://www.datypic.com/sc/xsd/t-xsd_NCName.html

**Comment by karinbredenberg** [comment]
Seems strange. That have work with other validators. We will look into it.

**Comment by hsilva-keep** [comment]
@karinbredenberg my apologies also for the late reply. 
In the ZIP file, you will find 2 xml files. One called "METS_correct.xml", which was produced by our SIP creation tool RODA-in. The other, called "METS_incorrect.xml", has the same content as the previous one with a single difference: all UUID's have their prefix changed from "uuid-" to "uuid:" (which causes, as I've mentioned before, validation failure).

[METS.zip](https://github.com/DLMArchivalStandardsBoard/CommonSpecification/files/1248244/METS.zip)

**Comment by karinbredenberg** [comment]
@hsilva-keep thank you. I now see the problem. 
There are different type of ID's and different rules should be used in the XML-document and also described in the Common specification for packages. Using the [http://dasboard.eu/images/Specifications/SIP/General_SIP-Specification_v1.4.pdf](url) specification from the DAS Board for the SIP for the elements where the ID rules are correct. (So there will be an update made in the Common specification document in chapter 5.2.)
The OBJID and that has not really a given rule for how it should look since it is a string in the schema and the value you give depends on the naming convention you have in your system used for the whole objects you are storing. (See page 15 in the SIP specification). 
For the attribute ID used for referencing in the XML document you should have a prefix and then an identification code. (See page 30 in the SIP specification) In the element "identification of the file" this is stated, "suggested use is  a prefix “ID” directly followed by an UUID or GUID or own local identification code. ID follows the rules of the xml attribute XML:ID." 
So do as you did in the one you called correct. Go without the ":" and even without the "-" and just use "ID" as suggested.

**Comment by jmaferreira** [comment]
I would suggest making "UUID" lowercase. Example:

uuid-5d39...

**Comment by karinbredenberg** [comment]
From the current Swedish profile:
```
<p xmlns="http://www.w3.org/1999/xhtml">The attribute ID can be used, it's the identification when referencing itself in the METS-document</p>
<p xmlns="http://www.w3.org/1999/xhtml">ID is set with for example UUID or GUID and starts with an added "ID"</p>
```
Example:
ID="ID550e8400-e29b-41d4-a716-44665544000z"

**Comment by kuldaraas** [comment]
Hopefully solved in ver 1.1
